security.rst

```diff
@@ -810,6 +810,12 @@ Prepending the path with ``^`` means that only URLs *beginning* with the
 pattern are matched. For example, a path of ``/admin`` (without the ``^``)
 would match ``/admin/foo`` but would also match URLs like ``/foo/admin``.
 
+.. note::
+
+    If you have multiple firewalls, you must use the internal pattern
```

Comment:
I honestly do not understand what "internal pattern" means. Can you elaborate a bit what you mean?

Reply:
I mean, if the user has multiple firewalls, then it should use the URL pattern in his firewall, and it's not enough to create a pattern in access_control section.
Need more explanation?